docs/user-guide/security-features.md

```diff
@@ -1,152 +1,152 @@
+When you select an environment, it automatically adopts the policy of the associated Cluster. For example, if critical-level vulnerabilities are blocked globally but allowed in the Cluster Security Policy, the Environment Security Policy will inherit this allowance. Consequently, critical-level vulnerabilities will also be allowed in the Environment Security Policy.
 
-Example-if you block critical severity levels in Global, then critical levels will be blocked in Cluster Security Policy. In case we change critical policy globally and allow it there, then it will be allowed in Cluster Security Policies too. But you can change these policies explicitly.
+However, you have the flexibility to make explicit changes to the policy if needed. This empowers you to customize the policy to align with specific requirements or preferences.
 
-If you want to block Critical Vulnerabilities in Global Security Policies but want to allow them in some clusters, then select your cluster and change the critical drop-down to allow. It will not affect the policy of other clusters and global also.
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/environment-security-policy.jpg)
 
-## Configure Environment Security Policy
+## Configure Application Security Policy
 
-Again we have three options to define a policy- Block, Allow, and Inherit.
+The Application Security Policy operates on a similar principle as other policies. However, in the Application Security Policy, the policy is determined by both the Environment option and the Application option.
 
-Environment Security Policy inherits the policy from Cluster Security Policy. Each level inherits the policy of its upper level.
+When modifying the policy within a development environment, the changes will be applied to all applications within that specific development environment. This means that any adjustments made to the policy settings will be consistently applied across all applications associated with that particular development environment.
 
-Select any environment here, you will find it is inheriting the policy of Cluster.
+This approach ensures uniformity and streamlined management of security policies within specific environments and their corresponding applications.
 
-Example- If you have blocked critical level vulnerabilities in Global Security Policy but allowed them in Cluster Security Policy, then Environment Security Policy will inherit the policy of cluster not global, Hence critical level vulnerabilities will be allowed in the Environment Security Policy.
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/application-security-policy.jpg)
 
-Though, You can change the policy explicitly.
+## Block or Allow Specific CVE Policies.
 
-## Configure Application Security Policy
+To block or allow specific Common Vulnerabilities and Exposures (CVE) policies, simply click on `Add CVE Policy`.
```

Comment:
To block or allow specific Common Vulnerabilities and Exposures (CVE) policies, simply click `Add CVE Policy`.